### DISCUSSION TITLE
Topolvm Scheduler in GKE

### DISCUSSION THREAD

**Discussion by Kavinraja-G** [topolvm/topolvm#713] [Q&A] [answered]
Is there any way we can enable the `topolvm-scheduler` in managed kubernetes clusters like GKE?

I'm facing errors in topolvm-scheduler pods --
```
2023-06-23T14:46:23.550012Z topolvm-scheduler-sdqwexc-sdwx topolvm-scheduler warning: "well: got signal" delay=5 signal="terminated"
```

**Comment by peng225** [comment, marked as answer]
Hi @Kavinraja-G 

> I'm facing errors in topolvm-scheduler pods --

I cannot guess what is going on from the provided log line.
Did you see the result of `kubectl describe pod`?

> Is there any way we can enable the topolvm-scheduler in managed kubernetes clusters like GKE?

Please see the following document.
https://github.com/topolvm/topolvm/tree/main/deploy#deploying-topolvm

In the managed Kubernetes environment, though, topolvm-scheduler may not work correctly because it is generally hard (or impssible?) to change the kube scheduler configuration.

**Reply by Kavinraja-G** [reply]
Got it, let me bug the provider and see if we can configure the `kube-scheduler`. Better, we can add a warning in the readme for managed env users.

**Comment by imul-p** [comment]
@Kavinraja-G, did you ever hear back regarding this error on GKE? I am looking at deploying the scheduler on EKS and am having the exact same error presented when the deployment starts.

**Reply by Kavinraja-G** [reply]
@imul-p Nope, GKE doesn't support changing scheduler configs. I think it will be the same case for EKS as well?
